extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -100,6 +74,7 @@ class AmpLightbox extends AMP.BaseElement {
         this.element.style.opacity = '';
       });
     }).then(() => {
+      this.build();
```

Comment:
Would it be better to do this up at the start of `activate()`?

I'm not sure this is not too late, but it would be more obvious to always do it before activation.